printer_copy_number/models/res_users.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    copies = fields.Integer(
```

Comment:
I would prefer this field to be named `print_copy_count` or something like that, as the users model is very generic, `copies` could be anything.